appeals/web/testing/app/fixtures/referencedata.js

```diff
@@ -76,6 +76,35 @@ export const appealData = {
 	],
 	localPlanningDepartment: 'Wiltshire Council',
 	lpaQuestionnaireId: null,
+	neighbouringSite: {
+		contacts: [
+			{
+				address: {
+					addressLine1: '21 The Pavement',
+					county: 'Wandsworth',
+					postCode: 'SW4 0HY'
+				},
+				contactId: 1016,
+				email: 'appellant@example.com',
```

Comment:
Yeah these to check, other than that looks good

Reply:
This is the data currently being returned by the API, only the address is actually used. Happy to strip these out if needed, but for now it's an accurate representation of the data from the endpoint.